lib/reservation.rb

```diff
@@ -0,0 +1,32 @@
+# Keeps track of 1 reservation
+require_relative 'room'
+require 'date'
+
+# module Hotel
+class Reservation
+  attr_reader :check_in_date, :check_out_date, :room_number, :nights_stayed
```

Comment:
`attr_reader`s are reserved for names of instance variables. `:nights_stayed` isn't the name of an instance var, it's the name of a method -- it doesn't need to be in this list!